ccl/multiregionccl: TestMultiRegionTenantRegions failed

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://mesolite.cluster.engflow.com/invocations/default/6bae938d-b6b5-4cd9-9511-f23fae2454b3?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [edf111e46e5105c0b21790536d706740e270aac1](https://github.com/cockroachdb/cockroach/commits/edf111e46e5105c0b21790536d706740e270aac1):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMultiRegionTenantRegions3010751859
    test_log_scope.go:76: use -show-logs to present logs inline
    multiregion_system_table_test.go:431: error executing query="ALTER DATABASE system DROP REGION \"us-east2\"" args=[]: ERROR: dial tcp 127.0.0.1:39913: connect: connection refused
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionTenantRegions3010751859
--- FAIL: TestMultiRegionTenantRegions (44.13s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-50691

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/93eeda7d-ae37-4b69-9ed1-4ab88d8a2aa7?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [52c0072ad3c389a0c3146891c48c07ae137450fe](https://github.com/cockroachdb/cockroach/commits/52c0072ad3c389a0c3146891c48c07ae137450fe):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMultiRegionTenantRegions3910386374
    test_log_scope.go:76: use -show-logs to present logs inline
    multiregion_system_table_test.go:382: error executing query="ALTER DATABASE system ADD REGION \"us-east2\"" args=[]: ERROR: dial tcp 127.0.0.1:35159: connect: connection refused
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionTenantRegions3910386374
--- FAIL: TestMultiRegionTenantRegions (917.09s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/d9636fb7-4e51-4d4f-b7a5-d17169b94c2d?testReportRun=2&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [a085c926b3ddeabb029d4ff5135d2165676ae7a2](https://github.com/cockroachdb/cockroach/commits/a085c926b3ddeabb029d4ff5135d2165676ae7a2):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMultiRegionTenantRegions4135853941
    test_log_scope.go:76: use -show-logs to present logs inline
    multiregion_system_table_test.go:437: error executing query="ALTER DATABASE system ADD REGION \"us-east3\"" args=[]: ERROR: dial tcp 127.0.0.1:44675: connect: connection refused
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionTenantRegions4135853941
--- FAIL: TestMultiRegionTenantRegions (55.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=2</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://tanzanite.cluster.engflow.com/invocations/default/3335ed0c-b437-4a74-8d36-b1d9fdc214d1?testReportRun=1&testReportShard=7&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [a085c926b3ddeabb029d4ff5135d2165676ae7a2](https://github.com/cockroachdb/cockroach/commits/a085c926b3ddeabb029d4ff5135d2165676ae7a2):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestMultiRegionTenantRegions713995241
    test_log_scope.go:76: use -show-logs to present logs inline
    multiregion_system_table_test.go:431: error executing query="ALTER DATABASE system DROP REGION \"us-east2\"" args=[]: ERROR: dial tcp 127.0.0.1:41705: connect: connection refused
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionTenantRegions713995241
--- FAIL: TestMultiRegionTenantRegions (49.22s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=7</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/multiregionccl.TestMultiRegionTenantRegions [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19730959?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/19730959?buildTab=artifacts#/) on master @ [ba8222d01c3ed85c34b7a05fbd58b87414be4225](https://github.com/cockroachdb/cockroach/commits/ba8222d01c3ed85c34b7a05fbd58b87414be4225):


```
=== RUN   TestMultiRegionTenantRegions
    test_log_scope.go:165: test logs captured to: /artifacts/tmp/_tmp/0af64d7faba6d19f64f9663e8459086e/logTestMultiRegionTenantRegions2854130586
    test_log_scope.go:76: use -show-logs to present logs inline
    sql_runner.go:317: error executing 'SELECT count(*) FROM system.sqlliveness WHERE crdb_region = 'us-east2' AND crdb_internal.sql_liveness_is_alive(session_id, true)': dial tcp 127.0.0.1:36869: connect: connection refused
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/0af64d7faba6d19f64f9663e8459086e/logTestMultiRegionTenantRegions2854130586
--- FAIL: TestMultiRegionTenantRegions (18.31s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionTenantRegions.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/multiregionccl/multiregion_system_table_test.go
+++ b/pkg/ccl/multiregionccl/multiregion_system_table_test.go
@@ -350,6 +350,7 @@ func TestMultiRegionTenantRegions(t *testing.T) {
 
 	// Shorten the sqlliveness TTL to speed up the test.
 	tenEast1SQLDB.Exec(t, "SET CLUSTER SETTING server.sqlliveness.ttl = '5s'")
+	tenEast1SQLDB.Exec(t, "SET CLUSTER SETTING server.sqlliveness.heartbeat = '1s'")
 
 	// Update system database with regions.
 	checkRegions := func(t *testing.T, regions ...string) {

